v3/src/utilities/translation/lang/en-US.json5

```diff
@@ -33,6 +33,10 @@
     "V3.Redo.caseTable.create": "Redo adding case table",
     "V3.Undo.caseTable.delete": "Undo deleting case table",
     "V3.Redo.caseTable.delete": "Redo deleting case table",
+    "V3.Undo.caseTable.show": "Undo showing case table",
+    "V3.Redo.caseTable.show": "Redo showing case table",
+    "V3.Undo.caseTable.hide": "Undo hiding case table",
+    "V3.Redo.caseTable.hide": "Redo hiding case table",
```

Comment:
V2 doesn't have strings for these?

Reply:
Turns out it had the strings for showing but not for hiding.